sensor/common/store/service/servicewrapper/wrap.go

```diff
@@ -0,0 +1,121 @@
+package servicewrapper
```

Comment:
Note to reviewers: this **isn't** new code. This was moved from `listener/resources/service.go`. Some struct/func names were changed.